### PR TITLE
Issue 141

### PR DESCRIPTION
**Pull request by fassko**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -280,4 +280,4 @@ DEPENDENCIES
   octokit
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```
Comment (BasThomas) [inline Gemfile.lock]: Why were these updated? 🤔

Comment (fassko) [reply Gemfile.lock]: Ok we can do it on master. Just when I wanted to run with this with Jekyll it yelled at me. :P

**Comment by fassko** [comment]
> This is looking awesome, thank again so much @fassko 🎉🎉🎉
> 
> Some thoughts:
> 
> * It would be nice if we can always leave a non-link between a name and article/forum post/whatever; two links side-by-side are hard to parse as two separate ones.

Clear.

> * We should avoid abbreviations like "PR", as per [the guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md#-writing-style-guide) [](/SwiftWeekly/swiftweekly.github.io/blob/HEAD@{2019-08-21T21:22:04Z}/CONTRIBUTING.md#-writing-style-guide).

OK, thanks. Somehow missed this.

**Comment by fassko** [comment]
I removed empty sections. If it seems fine you can merge it. I'm done with this as we need to publish this today.
